utils/chains.ts

```diff
@@ -135,4 +160,16 @@ export default [
     collectionSetId: process.env.NEXT_PUBLIC_BSC_COLLECTION_SET_ID,
     community: process.env.NEXT_PUBLIC_BSC_COMMUNITY,
   },
+  {
+    ...base,
+    lightIconUrl: '/icons/base-icon-dark.svg',
+    darkIconUrl: '/icons/base-icon-light.svg',
+    reservoirBaseUrl: 'https://api-base.reservoir.tools',
+    proxyApi: '/api/reservoir/base',
+    routePrefix: 'base',
+    apiKey: process.env.BASE_RESERVOIR_API_KEY,
+    coingeckoId: 'ethereum',
```

Comment:
@lunarmayor I couldn't find a `coingeckoId` for base. We have the zora chain configured to use 'ethereum', so figgured I'd do that here too 🤷‍♂️